README.md

```diff
@@ -33,7 +33,7 @@ Roadmap:
 
 ```python
 from pyspark.ml.linalg import DenseMetrix
-from rikai.vision import Image, BBox
+from rikai.types.vision import Image, Box2d
```

Comment:
should users just do from `rikai.types import Image, Box2d` ? I think Box2d is moved to geometry right? Users might find it hard to remember which package to import from?

Reply:
sure , will change that